README.md

```diff
@@ -187,19 +187,21 @@ spec:
     ## Audit Config
     # https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-policy
     # Define basic arguments for auditing log files. If needed, more complex flags
-    # can be set via 'Args' field manually.
+    # can be set via 'Args' field manually. Non-empty 'policy' field is required
+    # for logging. If 'logOutputVolumeClaim' is empty log to stdout otherwise it
+    # is a name of user defined PersistentVolumeClaim resourse.
```

Comment:
```suggestion
    # can be set via 'Args' field manually. 
    # Non-empty 'policy' field is mandatory to enable audit logging.
    # If 'logOutputVolumeClaim' is empty the audit log is printed to stdout, otherwise it
    # point to the user defined PersistentVolumeClaim resource name.
```